modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/FDS/fdsUtils/fdsRenderers.tsx

```diff
@@ -16,7 +16,18 @@ export function itemMethodRenderer({
 }: {
 	itemData: {httpMethod: {name: string}};
 }) {
-	return <ClayLabel displayType="info">{itemData.httpMethod.name}</ClayLabel>;
+	if (itemData.httpMethod.name === 'post') {
+		return (
+			<ClayLabel displayType="success">
```

Comment:
How do you see creating a function here to set the displayType?

Something like `<ClayLabel displayType={getDisplayType()}>`

This getDisplayType() returning "success" or "info" depending on the httpMethod.

Reply:
Make a lot of sense @jaimelr10! :smile: